databuilder/contracts/contracts.py

```diff
@@ -6,6 +6,8 @@
 class PatientDemographics(TableContract):
     """Provides demographic information about patients."""
 
+    _name = "patient_demographics"
```

Comment:
So that it doesn't clash with a possible field name.

Reply:
Aaaah. Yeah, okay. I thought that it was for filtering it out -- I hadn't thought it might actually stomp on something that was needed for a field. I don't love it, but fair enough.